requirements-dev.txt

```
@@ -0,0 +1,4 @@
+-r requirements.txt
+flake8==3.7.7
+mock==2.0.0
```

The latest version of mock on PyPI is 4.0.2

```suggestion
mock==4.0.2
```

---

I don't think I actually ended up using it after all!